ui/js/values.js

```diff
@@ -172,6 +172,52 @@ treeherder.value("phAlertResolutionMap", {
     DUPLICATE: 5
 });
 
+treeherder.value("phTalosDocumentationMap", {
```

Comment:
It's not 100% clear to me which parts are hacky here -- this information needs to be maintained somewhere, and I'm not sure if there's a better way of doing that than these sort of mappings. A purist would probably say that this information should live in a database somewhere, but that would bring complexities and complications of its own -- I'm not sure if that's the right trade off. Let's see how this goes and file bugs in the future if we find out that this method is a pain.